eventuate-crdt/src/it/scala/com/rbmhtechnology/eventuate/crdt/CRDTSerializerSpec.scala

```diff
@@ -93,12 +118,24 @@ class CRDTSerializerSpec extends WordSpec with Matchers with BeforeAndAfterAll {
 
       serialization.deserialize(serialization.serialize(initial).get, classOf[ORCart[_]]).get should be(expected)
     }
+    "support RGArray serialization with default key serialization" in serializations.tail.foreach { serialization =>
```

Comment:
Please rename test, as it causes a [DuplicateNameException](https://travis-ci.org/RBMHTechnology/eventuate/jobs/291251606#L4026) (sorry, build issues still not fixed).

Reply:
After I fixed the duplicate name error locally and ran `CRDTSerializerSpec`, all `RGArray`-related tests failed. Maybe it's a good idea to run tests locally before pushing to Github e.g. with `it:testOnly com.rbmhtechnology.eventuate.crdt.CRDTSerializerSpec`.